package.json

```diff
@@ -324,7 +331,12 @@
         },
         {
           "command": "affinidiExplorer.viewProperties",
-          "when": "view == affinidiExplorer && viewItem == project || view == affinidiExplorer && viewItem == issuance || view == affinidiExplorer && viewItem == schema",
+          "when": "view == affinidiExplorer && viewItem == inactiveProject || viewItem == project || viewItem == schema || viewItem == issuance",
```

Comment:
Hmm, so you simplified it.. I'm up for it, but in the current state it won't work as expected.
It would only check "view" for the "inactiveProject" (first check) and not others.

We should either prepend `view == affinidiExplorer &&` to each condition or do something like this:
```
"when": "view == affinidiExplorer && (viewItem == inactiveProject || viewItem == project || viewItem == schema || viewItem == issuance)",
```

But I somewhat doubt if that would work.

Reply:
it doesn't work.. i replaced view condition back for all viewItems